lib/EarlyFunctionAnalysis/EarlyFunctionAnalysis.cpp

```diff
@@ -1827,10 +1929,16 @@ bool EarlyFunctionAnalysis::runOnModule(Module &M) {
                                  nullptr);
   FunctionAnalysisResults Properties(std::move(DefaultSummary));
 
+  // Cache pre-population
+  Properties.initializeCache(M, ABIRegisters, Binary);
```

Comment:
We don't call it cache anywhere else (I think).
What about simply `importModel`?
We could also consider embedding this functionality in the constructor.